bucket/msikombustor.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://geeks3d.com/furmark/kombustor/downloads/",
+    "description": "A standalone benchmark/stability test tool based on software from the maker of Furmark.",
+    "license": "Freeware",
+    "version": "4.1.0.0",
+    "url": "https://gpuscore.top/msi/MSI_Kombustor4_Setup_v4.0.1.0_x64.exe",
+    "hash": "2ff0043a3ba45b3a611d18d65002edaf90ddde9fd95b41889540746478985840",
+    "bin": "MSI-Kombustor-x64.exe",
+    "innosetup": true,
+    "shortcuts": [
+        [
+            "MSI-Kombustor-x64.exe",
+            "MSI Kombustor"
+        ]
+    ],
+    "checkver": {
+        "url": "https://geeks3d.com/furmark/kombustor/downloads/",
+        "regex": "MSI Kombustor ([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://gpuscore.top/msi/MSI_Kombustor4_Setup_v4.0.1.0_x64.exe"
```

Comment:
- Make it architecture specific
- Add hash extraction

![image](https://user-images.githubusercontent.com/13260377/65261089-c81b5800-db07-11e9-85d4-7747e979b22e.png)